src/main/java/commands/MarkCommand.java

```diff
@@ -0,0 +1,25 @@
+package commands;
+
+import functional.TaskList;
+import functional.Ui;
+import tasks.Task;
+
+public class MarkCommand extends commands.Command {
+
+    public MarkCommand() {
+        super();
+    }
+
+    public void execute(TaskList<Task> tasks, Ui ui, boolean marked, boolean load) throws functional.DukeException {
```

Comment:
Any reason why you throws functional.DukeException rather than just throws DukeException?